dist/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone API Project</title>
+<script defer src="./bundle.js"></script></head>
+<body>
+    <header>
+        <a href="#" class="img-container"><img src="./901a179b50b06fd77621.png" alt="logo" class="header-img"></a>
+
+        <nav >
```

Comment:
- [ ] Please, consider that with the `npm start` command, your app opens on the 8080 port like this:

![image](https://user-images.githubusercontent.com/49064106/189500925-4507a671-4828-4464-916c-f4bf68978ff1.png)

- [ ] Please, consider also that the live server is open in the same way on port 5500:
![image](https://user-images.githubusercontent.com/49064106/189500953-2ed9c9b6-1654-4335-85e5-7e488aa59452.png)

- I can view your app only via the demo live link, you shared in the README file (https://maaloum.github.io/into-film/dist/), so I have some comments, which I will write under the exact code.

Reply:
Hi @elmar8287. Thank you for the review. We are not clear about the way the deployed version is being rendered on your end because it is rendering just fine on our end. Please advice how we can rectify this.